tests/common/snappi_tests/traffic_flow_config.py

```diff
@@ -0,0 +1,46 @@
+"""
+The TrafficFlowConfig module allows for modular pass through of traffic params for all Snappi based tests.
+"""
+
+
+class TrafficFlowConfig():
+    def __init__(self):
+        """
+        Initialize the TrafficFlowConfig class
+
+        Params:
+            data_flow_config (dict): data traffic flow configuration
```

Comment:
It seems to me that data_flow_config, background_flow_config and pause_flow_config can use same keys, e.g., dur_sec, rate_percent, rate_pps. etc.

Reply:
yes